$OpenBSD: patch-default_build_properties,v 1.3 2013/05/17 20:14:49 jeremy Exp $

Reduce amount of memory used when launching jruby.  It doesn't
require 1G, and without this patch you can't build JRuby on a
system with only 1G of memory.

--- default.build.properties.orig	Thu May 16 11:51:15 2013
+++ default.build.properties	Thu May 16 13:22:26 2013
@@ -61,8 +61,8 @@ html.test.results.dir=${test.results.dir}/html
 test.coverage.results.dir=${test.results.dir}/coverage
 javac.version=1.6
 jruby.compile.memory=512M
-jruby.launch.memory=1024M
-jruby.test.memory=1024M
+jruby.launch.memory=512M
+jruby.test.memory=512M
 jruby.test.memory.permgen=512M
 jruby.test.jvm=java
 rake.args=
